test/plugin/test_qiskit_plugins.py

```diff
@@ -22,27 +22,23 @@
 SOFTWARE.
 """
 
-from qiskit import QuantumCircuit
-import numpy as np
 import random
-from qiskit.opflow import StateFn, X, Y, Z, I
 
-import torchquantum as tq
+import numpy as np
+from qiskit.quantum_info import Pauli, Statevector
 
-from torchquantum.plugin import op_history2qiskit, QiskitProcessor
+import torchquantum as tq
+from torchquantum.plugin import QiskitProcessor, op_history2qiskit
 from torchquantum.util import switch_little_big_endian_state
 
-import torch
-import pytest
-
 pauli_str_op_dict = {
-    "X": X,
-    "Y": Y,
-    "Z": Z,
-    "I": I,
+    "X": Pauli("X"),
+    "Y": Pauli("Y"),
+    "Z": Pauli("Z"),
+    "I": Pauli("I"),
 }
 
-@pytest.mark.skip
```

Comment:
Hi! Thanks so much for the PR! Could you re-add this line? We unfortunately need it for now to run on GH Actions.

Reply:
Thank you for the feedback! I put this back now.

Reply:
Perfect, thanks so much!